test/integ/mandelbrot/CMakeLists.txt

```diff
@@ -51,11 +51,12 @@ SET(_SOURCE_DIR "src/")
 # Add all the source files in all recursive subdirectories and group them accordingly.
 append_recursive_files_add_to_src_group("${_SOURCE_DIR}" "${_SOURCE_DIR}" "cpp" _FILES_SOURCE)
 
-INCLUDE_DIRECTORIES(
-    ${_INCLUDE_DIRECTORIES_PRIVATE}
```

Comment:
There are a lot of there `*_PRIVATE` variables disappearing. They are not needed anymore ?